conn2res/plotting.py

```diff
@@ -70,3 +135,77 @@ def plot_performance_curve(df, title, num=2, figsize=(12, 10), savefig=False, bl
         fig.savefig(fname=os.path.join(FIG_DIR, f'{title}_score.png'),
                     transparent=True, bbox_inches='tight', dpi=300)
     plt.show(block=block)
+
+
+def plot_time_series(x, feature_set='orig', idx_ts=None, idx_features=None, n_features=None,
+                     scaler=1, num=1, figsize=(12, 6), subplot=None, title=None, fname='time_course',
+                     legend_label='', savefig=False, block=True, **kwargs):
+    # transform data
+    x = transform_data(
+        x, feature_set, idx_features=idx_features, n_features=n_features, scaler=scaler, **kwargs)
```

Comment:
The idea is that we transform the data before plotting so that, for instance, we can define the features which we want to plot, get principal components of the data etc.